package.json

```diff
@@ -56,7 +57,6 @@
     "babel-jest": "^25.1.0",
     "babel-loader": "^8.0.6",
     "babel-plugin-syntax-async-functions": "^6.13.0",
-    "cacophony-types": "github:hardiesoft/cacophony-types#master",
```

Comment:
Should this be listed under hardiesoft?

Reply:
I think this was something I was experimenting with a long time ago, and somehow got merged in somewhere along the way.  It's not being used, so I removed it.